### DISCUSSION TITLE
GitHub Actions is currently disabled for your account.

### DISCUSSION THREAD

**Discussion by hiyangguo** [community/community#19015] [Actions]
![image](https://user-images.githubusercontent.com/15609339/174701719-5fd423e2-b047-4b2d-99d2-1eb1b88b82fc.png)

How can I enable GitHub Actions, It's very useful for me.

**Comment by melvincwng** [comment]
Hi @hiyangguo, if you are having issues with GitHub Actions being disabled on your account, please kindly submit a ticket to [GitHub Support](https://support.github.com/request) over here.

They usually respond within a few working days, after performing a check/investigation on why that setting was disabled.

Hope this clarifies.

**Reply by hiyangguo** [reply]
I am multiple submit  tickets for this problem, but up to now , I haven‘t get  any valuable help for me.So how can I get help ? thank you very much. @melvincwng

**Reply by crexlight** [reply]
2 weeks, nobody cares about my ticket.

**Comment by fedekunze** [comment]
@melvincwng we also had the same problem with our organization. We don't know why GitHub Actions were disabled and as @hiyangguo mentioned the error message is not helpful at all. We are counting on GA for releases and for our testing environments and we are currently stuck.

We started having this issue when we transferred the public repositories to our new organization name. We tried updating billing (adding card, changing to yearly billing, upgrading to Team plan and then to Enterprise plan), toggled account settings, updated GitHub actions org settings, and repository-specific GA settings, all without success.

We reached out to Github Support almost a week ago, through numerous accounts (mine and my co-founders included), but we haven't received a single response yet to the tickets we created.

**Reply by melvincwng** [reply]
@fedekunze, I understand your frustrations in your organization not being able to use Github Actions.

Based on my understanding from the community forums, there have been instances of others not being able to use GitHub Actions as well.

In fact, you can visit the discussion threads [here](https://github.community/t/actions-are-currently-disabled-for-your-account/176652) and [here](https://github.community/t/actions-are-currently-disabled-for-your-account-i-want-to-enable/173147).

Unfortunately, there's really nothing much we can do at the moment, other than waiting patiently for the investigation(s) by Github Support.

Please kindly wait while they get back to your support tickets. 

Hopefully, they are able to figure out what’s wrong and get your actions back up and running again.

**Reply by fedekunze** [reply]
I'm pretty sure this is a bug because we only have open-source, public repositories in our organization which should have GA support enabled by default as stated [here](https://github.com/features/actions) 

See also: https://github.com/github-community/community/discussions/17493

**Reply by melvincwng** [reply]
@fedekunze, seems like the Github Bug Report form has been fixed by the Github team. For context purposes, there was an issue with their bug report form a couple of days back.

Since that is the case, I suggest both of you to report it as a bug over [here](https://support.github.com/contact/bug-report).

**Reply by fedekunze** [reply]
Thanks @melvincwng I created a bug report. Would it be possible to connect with someone from Github Support or the GitHub Actions team to figure out what's happening? My [DMs on Twitter](https://twitter.com/fekunze) are open.

**Comment by lance6716** [comment]
A temporary walkaround: my fellow has been banned from Actions but I can use it, we found that if someone who is accessible to GitHub Actions adds a commit to the PR, the PR can trigger GitHub Actions. @fedekunze

**Reply by hiyangguo** [reply]
emm

**Comment by cirosantilli** [comment]
They replied to me after 42 days explaining the reason and we managed to sort it out: https://github.com/cirosantilli/china-dictatorship/issues/1330 I just wish they'd immediately explain the reason when they do the block, it was likely a manual block in my base.

**Comment by kinguser981** [comment]
running RDP on GitHub Actions Causing this issue.

**Comment by zainulnazir** [comment]
I also have the same issue, but I am unable to fix it. I removed the possible cause for the problem but looks like the issue's still there.

**Comment by kinguser981** [comment]
dont put comments below this post its not fixable!!!!
create new account . byeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee
